src/actions/heap/heap.ts

```diff
@@ -38,8 +38,13 @@ export class HeapAction extends Hub.Action {
     "https://heapanalytics.com/api/integrations/add_user_properties"
   static ADD_ACCOUNT_PROPERTIES_URL =
     "https://heapanalytics.com/api/add_account_properties"
+  static HEAP_TRACK_URL = "https://heapanalytics.com/api/track"
   static HEAP_LIBRARY = "looker"
   static ROWS_PER_BATCH = 1000
+  // TODO: remove me before GA
+  static HEAP_ENV_ID = 236035469
```

Comment:
are there any risks of exposing env ID and identity in a public repo? might be safer to pass these as env variables in the Dockerfile